File: tyrant/scala/delete/tyrant_delete.scala
// ----------------------------------------------------------------
/*
	mcachedb_delete.scala

					May/29/2012

*/
// ----------------------------------------------------------------
object	tyrant_delete
{
// ----------------------------------------------------------------
def main (args: scala.Array[String])
{
	println ("*** 開始 ***")

	val key_in = args(0)

	println (key_in)

	val server = "localhost"
	val port = 1978
	print (server + '\t')
	println (port)

	mcached_manipulate.delete_record_proc (key_in,server,port)

	println ("*** 終了 ***")
}

// ----------------------------------------------------------------
}
// ----------------------------------------------------------------

File: tyrant/erlang/delete/tyrant_delete.erl
#! /usr/bin/escript
%% -----------------------------------------------------------
%%	tyrant_delete.erl
%%
%%					May/13/2013
%% -----------------------------------------------------------
-module(socket).
%% -----------------------------------------------------------
main (A)->
	io:format ("*** 開始 ***\n"),
	[Key_in]=A,
	io:format (lists:concat (["Key_in = ",Key_in,"\n"])),
%%
	Host = "localhost",
	Port = 1978,
	{ok, Sock} = gen_tcp:connect(Host, Port,
                                 [binary, {packet, 0}]),
	Command =  lists:concat(["delete ",Key_in, "\r\n"]),
	ok = gen_tcp:send(Sock, Command),
	ok = gen_tcp:close(Sock),
%%
	io:format ("*** 終了 ***\n").
%% -----------------------------------------------------------
